metrics/density/docker_memory_usage.sh

```diff
@@ -280,6 +285,10 @@ EOF
 			"Result": $hypervisor_mem,
 			"Units" : "KB"
 		},
+		"virtiofsd": {
```

Comment:
Nit: Not inserted into the file alphabetically.

Reply:
should be "virtiofsds" here?